src/Autonomous.cpp

```diff
@@ -87,20 +79,21 @@ double transformAngle(double currAngle, double targetAngle)
 	return currAngle + ((absDist > (range / 2)) ? -sign * (range - absDist) : dist);
 }
 
-double getThetaVel(const PointXY &target, const pose_t &pose, double &thetaErr)
+double getThetaVel(const point_t &target, const pose_t &pose, double &thetaErr)
 {
-	double dx = target.x - pose[0];
-	double dy = target.y - pose[1];
+	double dx = target(0) - pose(0);
```

Comment:
I think with Eigen the "correct" way to access vector elements is with parentheses rather than brackets. Not sure why it worked before

Reply:
`point_t` is an `Eigen::Vector3d`, and Eigen supports `operator[]` for vectors. However, I agree that using parenthesis instead of brackets is better for consistency.